rpgJavaInterpreter-core/src/test/kotlin/com/smeup/rpgparser/smeup/PerfLoggingExample.kt

```diff
@@ -16,4 +16,36 @@ class PerfLoggingExample : MULANGTTest() {
         }
         executePgm(programName = "performance/MUTE10_10", systemInterface = si)
     }
+
+    @Test @Category(PerformanceTest::class)
+    fun testT10_A60_P02() {
+        val si = JavaSystemInterface().apply {
+            loggingConfiguration = consoleLoggingConfiguration(LogChannel.PERFORMANCE, LogChannel.ANALYTICS)
+        }
+        executePgm(programName = "smeup/T10_A60_P02", systemInterface = si)
+    }
+
+    @Test @Category(PerformanceTest::class)
+    fun testT10_A60_P03() {
+        val si = JavaSystemInterface().apply {
+            loggingConfiguration = consoleLoggingConfiguration(LogChannel.ANALYTICS)
+        }
+        executePgm(programName = "smeup/T10_A60_P03", systemInterface = si)
+    }
+
+    @Test @Category(PerformanceTest::class)
+    fun testT40_A20_P17() {
+        val si = JavaSystemInterface().apply {
+            loggingConfiguration = consoleLoggingConfiguration(LogChannel.ANALYTICS)
```

Comment:
In this test, there is no trace of the symbol table's get action, despite the DO statement continuously accessing the symbol table to evaluate the exit condition. If we want to be sure not to lose anything, we have to implement this logic in the SymbolTable.get and SymbolTable.set methods. You can do this in the concrete implementation com.smeup.rpgparser.interpreter.SymbolTable because com.smeup.rpgparser.experimental.SymbolTable is deprecated and was nothing more than an experiment.